packages/flutter/lib/src/widgets/basic.dart

```diff
@@ -1999,6 +2018,9 @@ class Flex extends MultiChildRenderObjectWidget {
   final Axis direction;
 
   /// How the children should be placed along the main axis.
+  ///
+  /// For example, [MainAxisAlignment.start], the default, places the children
+  /// at the start (e.g., the top for a [Column]) of the main axis.
```

Comment:
I was trying to avoid the reading direction issue, but that's not really worth it.